packages/frontend-2/pages/server-management/index.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink
+        :to="'/server-management'"
+        name="Server Management"
+      ></HeaderNavLink>
+    </Portal>
+    <div
+      class="flex flex-col md:flex-row space-y-2 space-x-2 justify-between mb-4 md:items-center"
+    >
+      <div>
+        <h5 class="h4 font-bold">Your server at a glance</h5>
+      </div>
+    </div>
+
+    <div class="grid grid-cols-1 md:grid-cols-2 gap-6 mt-10">
+      <Card :server-info="serverData" @cta-clicked="showDialog = true" />
+
+      <SettingsDialog
+        ref="settingsDialog"
+        v-model:open="showDialog"
+        title="Edit Settings"
+        :buttons="[
+          {
+            text: 'Cancel',
+            props: { color: 'secondary', fullWidth: true, outline: true },
+            onClick: closeDialog
+          },
+          {
+            text: 'Save',
+            props: { color: 'primary', fullWidth: true, outline: false },
+            onClick: saveSettings
+          }
+        ]"
+        @server-info-updated="fetchData"
+      />
+
+      <Card :server-info="userData" />
+      <Card :server-info="projectData" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { graphql } from '~~/lib/common/generated/gql'
+import { useQuery } from '@vue/apollo-composable'
+import { ref, computed, onMounted, reactive } from 'vue'
+import Card, { CardInfo } from '../../components/server-management/Card.vue'
```

Comment:
Remove all manual imports of local vue components in all files, Nuxt auto-imports local components.

https://nuxt.com/docs/guide/concepts/auto-imports
https://nuxt.com/docs/guide/directory-structure/components#component-names

Reply:
+ Make sure you always use absolute path imports